modules/ROOT/pages/deploying-receive-endpoints.adoc

```diff
@@ -1,55 +1,69 @@
 = Deploying AS2, HTTP, and HTTPS Receive Endpoints
 
-You can deploy AS2 receive, HTTP receive, and HTTPS receive endpoints on a public or private port, or on-premises.
+You can deploy AS2 receive, HTTP receive, and HTTPS receive endpoints on a public port, private port, or on-premises.
 
 == Deploy to CloudHub on a Public Port
 
-When an AS2, HTTP, or HTTPS receive endpoint is deployed with the public port configuration enabled, the endpoint application is deployed on the public HTTPS port (`8082`), with the traffic routed through the xref:runtime-manager::dedicated-load-balancer-tutorial#shared-load-balancers[shared load balancer]. The value in the URL field is automatically generated when you deploy a message flow to CloudHub that uses this endpoint.
+When an AS2, HTTP, or HTTPS receive endpoint is deployed with the public port configuration enabled, the endpoint application is deployed on the public HTTPS port (`8082`), with the traffic routed through the xref:runtime-manager::dedicated-load-balancer-tutorial.adoc#shared-load-balancers.adoc[shared load balancer]. The value in the URL field is automatically generated when you deploy a message flow to CloudHub that uses this endpoint.
 
-You can view this URL by either selecting the receive endpoint from your deployed message flow or clicking the endpoint name from the endpoints list. You can share this URL with the partners that send AS2, HTTP, or HTTPS messages to this endpoint.
+You can view the runtime application name and external URL for a deployed endpoint from the endpoint details page. For example:
+
+image::pm-receive-endpoint-url.png[Example of a runtime application name, internal URL, and external URL]
```

Comment:
should this image be under the images folder? i think right now it isnt

Reply:
It is there; pm-receive-endpoint.url